scripts/generators/stateless_validation_helper_generator.py

```diff
@@ -511,6 +509,16 @@ def generateSource(self):
             prototype = prototype.replace(')', ', const ErrorObject& error_obj)')
             out.append(f'bool StatelessValidation::PreCallValidate{prototype} const {{\n')
             out.append('    bool skip = false;\n')
+            # For vkCreateDevice, the extensions member has already been set up properly
+            # for other VkPhysicalDevice calls, we need to use their supported extensions rather
+            # than the extensions members, which is how the VkInstance was configured.
+            if command.params[0].type == 'VkPhysicalDevice' and command.name != 'vkCreateDevice':
+                out.append('''
+                    const auto &physdev_extensions = physical_device_extensions.at(physicalDevice);
+                    stateless::State state(*this, error_obj, physdev_extensions, IsExtEnabled(physdev_extensions.vk_khr_maintenance5));
```

Comment:
```suggestion
                    stateless::State state(*this, error_obj, physdev_extensions);
```

then go `ignore_unknown_enums(IsExtEnabled(physdev_extensions.vk_khr_maintenance5))`

Reply:
I "think" it might be because we want all device level things to not be ignored... ya, close this if you thought of this already

Reply:
Yes, this suggestion won't work because then all device level checking would be ignored too. There's not an easy way to make a separate constructor